Crash in the web version

Install the latest web version from Github. Change Avoid options when building a route => Crash.

```
02-14 21:25:19.076 E/AndroidRuntime(14457): FATAL EXCEPTION: main
02-14 21:25:19.076 E/AndroidRuntime(14457): Process: app.organicmaps.web, PID: 14457
02-14 21:25:19.076 E/AndroidRuntime(14457): android.content.ActivityNotFoundException: Unable to find explicit activity class {app.organicmaps/app.organicmaps.settings.DrivingOptionsActivity}; have you declared this activity in your AndroidManifest.xml?
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2012)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1675)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.Activity.startActivityForResult(Activity.java:4602)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:780)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.Activity.startActivityForResult(Activity.java:4560)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:761)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.Activity.startActivity(Activity.java:4921)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.Activity.startActivity(Activity.java:4889)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at androidx.preference.Preference.performClick(Preference.java:1215)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at androidx.preference.Preference.performClick(Preference.java:1182)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at androidx.preference.Preference$1.onClick(Preference.java:182)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.view.View.performClick(View.java:6615)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.view.View.performClickInternal(View.java:6587)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.view.View.access$3500(View.java:784)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.view.View$PerformClick.run(View.java:26048)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.os.Handler.handleCallback(Handler.java:873)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.os.Handler.dispatchMessage(Handler.java:99)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.os.Looper.loop(Looper.java:205)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at android.app.ActivityThread.main(ActivityThread.java:6991)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at java.lang.reflect.Method.invoke(Native Method)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
02-14 21:25:19.076 E/AndroidRuntime(14457): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:884)
```

```
Видимо смена апп айди креш вскрыла?

02-14 19:21:39.101 E/AndroidRuntime(28178): java.lang.SecurityException: Permission Denial: starting Intent { cmp=app.organicmaps/.settings.DrivingOptionsActivity (has extras) } from ProcessRecord{2f08b58 28178:app.organicmaps.web/u0a60} (pid=28178, uid=10060) not exported from uid 10409
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.os.Parcel.createException(Parcel.java:1950)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.os.Parcel.readException(Parcel.java:1918)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.os.Parcel.readException(Parcel.java:1868)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.IActivityManager$Stub$Proxy.startActivity(IActivityManager.java:3634)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1671)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.Activity.startActivityForResult(Activity.java:4602)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:780)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.Activity.startActivityForResult(Activity.java:4560)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:761)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.Activity.startActivity(Activity.java:4921)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.Activity.startActivity(Activity.java:4889)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at androidx.preference.Preference.performClick(Preference.java:1215)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at androidx.preference.Preference.performClick(Preference.java:1182)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at androidx.preference.Preference$1.onClick(Preference.java:182)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.view.View.performClick(View.java:6615)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.view.View.performClickInternal(View.java:6587)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.view.View.access$3500(View.java:784)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.view.View$PerformClick.run(View.java:26048)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.os.Handler.handleCallback(Handler.java:873)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.os.Handler.dispatchMessage(Handler.java:99)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.os.Looper.loop(Looper.java:205)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at android.app.ActivityThread.main(ActivityThread.java:6991)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at java.lang.reflect.Method.invoke(Native Method)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:884)
02-14 19:21:39.101 E/AndroidRuntime(28178): Caused by: android.os.RemoteException: Remote stack trace:
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.server.am.ActivityStackSupervisor.checkStartAnyActivityPermission(ActivityStackSupervisor.java:1860)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.server.am.ActivityStarter.startActivity(ActivityStarter.java:750)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.server.am.ActivityStarter.startActivity(ActivityStarter.java:569)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.server.am.ActivityStarter.startActivityMayWait(ActivityStarter.java:1263)
02-14 19:21:39.101 E/AndroidRuntime(28178): 	at com.android.server.am.ActivityStarter.execute(ActivityStarter.java:507)
```

--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -258,7 +258,6 @@ android {
       jniDebuggable true           // Enable jni debug build
       zipAlignEnabled true
       signingConfig signingConfigs.debug
-      resValue 'string', 'app_id', android.defaultConfig.applicationId + applicationIdSuffix
       resValue 'string', 'app_name', 'Debug Organic Maps'
       // Do not generate separate debug symbols for debug apps, because we don't distribute them.
       ndk.debugSymbolLevel = 'none'
@@ -279,7 +278,6 @@ android {
       // Includes the default ProGuard rules files that are packaged with the Android Gradle plugin.
       // To learn more, go to the documentation section about R8 configuration files.
       proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-      resValue 'string', 'app_id', android.defaultConfig.applicationId
       resValue 'string', 'app_name', project.ext.appName
       // Full size symbols are too big for Google, 217mb aab vs 95mb.
       ndk.debugSymbolLevel = 'symbol_table'
@@ -301,7 +299,6 @@ android {
       // To learn more, go to the documentation section about R8 configuration files.
       proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
       matchingFallbacks = ['debug', 'release']
-      resValue 'string', 'app_id', android.defaultConfig.applicationId + applicationIdSuffix
       resValue 'string', 'app_name', 'Beta Organic Maps'
       // Full size symbols are too big for Google, 217mb aab vs 95mb.
       ndk.debugSymbolLevel = 'symbol_table'
@@ -417,6 +414,7 @@ android.applicationVariants.all { variant ->
   variant.buildConfigField 'String', 'FILE_PROVIDER_AUTHORITY', authority
   def flavor = variant.getMergedFlavor()
   flavor.manifestPlaceholders += [FILE_PROVIDER_PLACEHOLDER : authorityValue]
+  variant.resValue 'string', 'app_id', variant.applicationId
 }
 
 task prepareGoogleReleaseListing {

